### PR TITLE
setup-chainctl: provide defaults for env and audience

### PR DESCRIPTION
**Pull request by imjasonh**
Also use `https://dl.enforce.dev` instead of GCS.

### CODE REVIEW

File: setup-chainctl/README.md [hunk 1]
```diff
@@ -9,14 +9,14 @@ and authenticates with it using identity tokens.
...
+    # binary from.
+    # Optional (default is enforce.dev)
+    environment: enforce.dev
     # audience is the identity token audience to use when creating an identity
     # token to authenticate with Chainguard, it is required and has no default
     # (for now).
-    # Required.
-    audience: oscar-the-grouch
+    # Optional (default is https://issuer.enforce.dev)
+    audience: https://issuer.enforce.dev
```
Comment (mattmoor) [inline setup-chainctl/README.md:19]: Deprecate this and default to `issuer.{env}` like `dl.{env}`?

File: setup-chainctl/README.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ permissions:
 steps:
 - uses: chainguard-dev/actions/setup-chainctl@main
   with:
-    environment: big-bird
-    audience: elmo
+    environment: example.com
+    audience: some-other-audience
```
Comment (mattmoor) [inline setup-chainctl/README.md:36]: I'd make the sample just not specify these, since it is really only relevant to Chainguard folks (now)

**Review by mattmoor — APPROVED** [review]
Thanks, this is a great change 🤩 .  Let's stage the internal change and time this around our next push?

File: setup-chainctl/README.md (hunk 1 above)
Comment (imjasonh) [reply setup-chainctl/README.md:19]: There might still be value in configuring the audience, even when I want to download chainctl from dl.enforce.dev. I can make it default to issuer.$ENV though.

**Review by cpanato — APPROVED** [review]

**Pull request merged**